File: Assignment 3/Files/Student.py
from Person import Person

class Student:
    def __init__(self, newName="none", address="none", phone="999-999-9999", year = 1999):
        self.name = newName
        self.address = address
        self.phone = phone
        self.year = year 

    def getGraduationYear(self) -> int:
        return (self.year)
    
    def setGraduationYear(self, year):
        self.year = year

    def getName(self):
        return(self.name)

File: Assignment 4/Files/Student.java
public class Student extends Person{
    String name;
    String address;
    String phone;
    int year; 

   

    public Student(String NewName, String address, String phone, int year){
        super(NewName, address, phone);

        this.year = year;

    }

    public String toString(){
        return name + " " + address + " " + phone + " " + year; 
    }

    int getGraduationYear(){
        return(year);
    }

    void setGraduationYear(int year){
        this.year = year;
    }


}
